arangodb-net-standard/AdminApi/Models/ResponseBase.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Net;
+
+namespace ArangoDBNetStandard.AdminApi.Models
+{
+    /// <summary>
+    /// Represents a common response class for Admin API operations.
+    /// </summary>
+    public class ResponseBase
+    {
+        /// <summary>
+        /// Indicates whether an error occurred
```

Comment:
Consider rephrasing the summary to remove any ambiguity. For example:

```
/// Always false.
/// </summary>
/// <remarks>
/// To handle errors, catch <see cref="ApiErrorException"/>
/// thrown by API client methods.
/// </remarks>
```

It will be handy for anyone not familiar with the documentation about error handling available in the readme: https://github.com/ArangoDB-Community/arangodb-net-standard#api-errors

Reply:
Fixed